src/VisualStudio/Core/Def/TaskList/ExternalErrorDiagnosticUpdateSource.cs

```diff
@@ -365,16 +365,6 @@ internal void OnSolutionBuildCompleted()
                     return;
                 }
 
-                // Explicitly start solution crawler if it didn't start yet. since solution crawler is lazy, 
-                // user might have built solution before workspace fires its first event yet (which is when solution crawler is initialized)
-                // here we give initializeLazily: false so that solution crawler is fully initialized when we do de-dup live and build errors,
-                // otherwise, we will think none of error we have here belong to live errors since diagnostic service is not initialized yet.
-                if (_diagnosticService.GlobalOptions.GetOption(SolutionCrawlerRegistrationService.EnableSolutionCrawler))
```

Comment:
Wonder what purpose is served by `ExternalErrorDiagnosticUpdateSource` in lsp-pull only world. I thought it had two purposes - build/live de-duping and reporting build-only diagnostics, neither of which are owned by us in lsp-pull diagnostics world.

Reply:
i wanted to check on that with yuo.  i'm totally good with removing this :)  i just wanted certain if there was something i was missing here.